CHANGELOG.md

```diff
@@ -49,10 +50,6 @@
 - Initial example
 - Changelog #7
 
-### Fixed
-
-- Remove hardcoded nomad provider #6
-
```

Comment:
This shouldn't be removed, it's a part of the template's changelog, isn't it? (I might be wrong here)

Reply:
Yes, this should have been part of the minio changelog. It's on its way back

Reply:
Nice :D Think that's everything for me, rest looks good!